scripts/mumble-version.py

```diff
@@ -101,17 +101,18 @@ def readProjectVersion():
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-n', '--newline', action = "store_true", help = 'Break line after printing version')
-    parser.add_argument('-p', '--project', action = "store_true", help = 'Print CMake project version')
+    parser.add_argument('-f', '--format', choices = ['full', 'version', 'suffix'], help = 'Output format')
```

Comment:
```suggestion
    parser.add_argument('-f', '--format', choices = ['full', 'version', 'suffix'], help = 'Output format', default='version')
```

Reply:
Are you suggesting to explicitly set the default? If that's the case, it should be `default = 'full'`.

Reply:
Yes I am, but I think that the default should be `'version'`. That's probably the version that most folks have in mind when thinking about a version of Mumble :thinking:

Reply:
We would have to change all CI scripts though, as they assume the full version is the default.

Reply:
Considering that by default the release type is set to snapshot we should probably leave `full` as default, as it identifies the specific commit.

Reply:
Hm yeah fair enough :+1: